sanic/app.py

```diff
@@ -50,13 +50,18 @@ def __init__(self, name=None, router=None, error_handler=None):
         self.debug = None
         self.sock = None
         self.listeners = defaultdict(list)
+        self.is_running = False
 
         # Register alternative method names
         self.go_fast = self.run
 
     @property
     def loop(self):
         """Synonymous with asyncio.get_event_loop()."""
+        if not self.is_running:
```

Comment:
Can we add a boolean `self.is_async` or something, then say `if not self.is_running or self.is_async` here? Then when using the run async option users can still call `app.loop` if wanted, which shouldn't be an issue.